dom.go

```diff
@@ -268,8 +271,16 @@ func (h *HTML) Restore(old ComponentOrHTML) {
 // function is not used directly but rather the elem subpackage (which is type
 // safe) is used instead.
 func Tag(tag string, m ...MarkupOrComponentOrHTML) *HTML {
+	return NamespacedTag(``, tag, m...)
```

Comment:
Thank you for the brain-dump :wink:

My preference has always been to be explicit about where escape sequences or interpolation are expected, and I expect the parser to be simpler for raw strings, however consistency is absolutely a good enough reason to prefer one form from mostly equivalent options.

I must admit though that ergonomics does play a part in my preference - my pinky gets tired these days, and one keypress is less strain than two.  I'll have to consider before applying this to my personal code, as I write more Go.  Perhaps I'll try a keycode mod.

In any case, thanks for the discussion, change is inbound.